### DISCUSSION TITLE
Facing problems while using torch-geometric libraries in google0colab

### DISCUSSION THREAD

**Discussion by RaM2VaR** [pyg-team/pytorch_geometric#4558] [Q&A]
When i try to execute my code following error is encountered: tcmalloc: large alloc 139766055411712 bytes. Previous the code is used run on google-colab. Recently, I am facing this problem. 

Following libraries are used: torch-scatter torch-sparse torch-cluster torch-spline-conv torch-geometric

Error: 

tcmalloc: large alloc 139770292150272 bytes == (nil) @  0x7f1ecfe33887 0x7f1ece729c29 0x7f1ece72aafb 0x7f1ece72b20c 0x7f1e489ba028 0x7f1e489bdee8 0x7f1e0b0b49aa 0x7f1e0b0b0352 0x7f1ed007e8d3 0x7f1ed008339f 0x7f1ecfb7416f 0x7f1ed008296a 0x7f1ecf5eaf96 0x7f1ecfb7416f 0x7f1ecfb741ff 0x7f1ecf5eb745 0x7f1ecf5eb051 0x7f1ec82b894a 0x59371f 0x515244 0x549e0e 0x4bcb19 0x532b86 0x594a96 0x548cc1 0x51566f 0x593dd7 0x548ae9 0x51566f 0x549576 0x604173
terminate called after throwing an instance of 'std::bad_alloc'
  what():  std::bad_alloc

**Comment by rusty1s** [comment]
Please see https://github.com/pyg-team/pytorch_geometric/issues/4539.
